### ISSUE TITLE
Game saves are off.  

### ISSUE THREAD

**Issue by blakedunc235** [xerpi/realboy-vita#3]
In a couple games it's pretty hit or miss on whether or not emulator will actually save. In Pokemon Yellow I would save  then play for another 20 minutes and save again and when I come back to the game later only the first Dave is still there.

**Comment by Rubix117** [comment]
I was having trouble saving, I resolved this by placing the rom in the saves directory: ux0:data/realboy/saves. Where is your .gb file?

**Comment by Rubix117** [comment]
Disregard my last post. I am still having save issues.

**Comment by xerpi** [comment]
I've tried with Super Mario Bros. Deluxe and it works fine, can you tell the games you are using?

**Comment by Rubix117** [comment]
Trying it with pokemon. It seems like small intermittent failure. I have stated saving a couple of times to guarantee success.

**Comment by blakedunc235** [comment]
Yeah I usually save about 3 times and sometimes it still doesn't work. I might Bump it up to 10. Still not a deal breaker, I love being able to use the joystick instead of the directional pad to move around. Overall great work!!!

**Comment by xerpi** [comment]
I've tried quite a few times and all of them have worked.
Try with https://github.com/xerpi/realboy-vita/releases/tag/1.2

**Comment by Drommedhar** [comment]
Even 1.2 has this bug on my side. Tested with Pokémon silver. First Save worked but the rest after it did not. Hope there is a way to fix this. Otherwise Great work. 
Edit:
I have the Game in "ux0:/Homebrew/GB Roms/xxx.gbc"
Maybe it's something with the path?

**Comment by bioschaf** [comment]
I can confirm this bug with 1.2 and various games. Seems like the first save is generated as file in ux0:data/realboy/saves but then never overridden with consecutive saves

**Comment by yosh778** [comment]
I also experienced this on rejuvenate before once I got to Mt. Moon on Pokemon Yellow, my saved corrupted itself or disappeared somehow. I would suggest to keep a backup of old saves before overwriting maybe for now, could be handy :)

**Comment by blakedunc235** [comment]
I upgraded to 1.2 but I am still running into the same problem. For now on I'll just play a little, save then reset
